### ISSUE TITLE
Not work in IDEA 2021.2.2

### ISSUE THREAD

**Issue by way-zer** [syuchan1005/GitPrefix#32]
**Describe the bug**

**StackTrace**
```text
Cannot create listener

com.intellij.diagnostic.PluginException: Cannot create class com.github.syuchan1005.gitprefix.git.injector.GitInjectorManager (classloader=PluginClassLoader(plugin=PluginDescriptor(name=GitPrefix, id=com.github.syuchan1005.emojiprefix, descriptorPath=plugin.xml, path=E:\JetBrains-ToolBox\apps\IDEA-U\ch-0\212.5284.40.plugins\GitPrefix.jar, version=2.1.4, package=null), packagePrefix=null, instanceId=28, state=active))
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClass(ComponentManagerImpl.kt:870)
	at com.intellij.serviceContainer.ComponentManagerImpl.createListener(ComponentManagerImpl.kt:908)
	at com.intellij.util.messages.impl.CompositeMessageBus.subscribeLazyListeners(CompositeMessageBus.java:177)
	at com.intellij.util.messages.impl.CompositeMessageBus.doComputeSubscribers(CompositeMessageBus.java:147)
	at com.intellij.util.messages.impl.MessageBusImpl.computeSubscribers(MessageBusImpl.java:348)
	at com.intellij.util.messages.impl.CompositeMessageBus.computeSubscribers(CompositeMessageBus.java:141)
	at com.intellij.util.messages.impl.CompositeMessageBus$ToDirectChildrenMessagePublisher.lambda$publish$0(CompositeMessageBus.java:103)
	at java.base/java.util.concurrent.ConcurrentHashMap.computeIfAbsent(ConcurrentHashMap.java:1705)
	at com.intellij.util.messages.impl.CompositeMessageBus$ToDirectChildrenMessagePublisher.publish(CompositeMessageBus.java:103)
	at com.intellij.util.messages.impl.MessageBusImpl$MessagePublisher.invoke(MessageBusImpl.java:175)
	at com.sun.proxy.$Proxy38.appFrameCreated(Unknown Source)
	at com.intellij.idea.IdeStarter.openProjectIfNeeded(IdeStarter.kt:98)
	at com.intellij.idea.IdeStarter.main(IdeStarter.kt:77)
	at com.intellij.idea.ApplicationLoader$startApp$3.run(ApplicationLoader.kt:185)
	at java.base/java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:783)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1742)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1728)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool.awaitQuiescence(ForkJoinPool.java:2984)
	at com.intellij.idea.StartupUtil.start(StartupUtil.java:294)
	at com.intellij.idea.Main.bootstrap(Main.java:123)
	at com.intellij.idea.Main.main(Main.java:84)
Caused by: java.lang.IllegalAccessException: no such constructor: com.github.syuchan1005.gitprefix.git.injector.GitInjectorManager.<init>()void/newInvokeSpecial
	at java.base/java.lang.invoke.MemberName.makeAccessException(MemberName.java:959)
	at java.base/java.lang.invoke.MemberName$Factory.resolveOrFail(MemberName.java:1101)
	at java.base/java.lang.invoke.MethodHandles$Lookup.resolveOrFail(MethodHandles.java:2030)
	at java.base/java.lang.invoke.MethodHandles$Lookup.findConstructor(MethodHandles.java:1264)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClass(ComponentManagerImpl.kt:830)
	... 22 more
Caused by: java.lang.NoClassDefFoundError: javassist/ClassPath
	at java.base/java.lang.invoke.MethodHandleNatives.resolve(Native Method)
	at java.base/java.lang.invoke.MemberName$Factory.resolve(MemberName.java:1070)
	at java.base/java.lang.invoke.MemberName$Factory.resolveOrFail(MemberName.java:1098)
	... 25 more
Caused by: java.lang.ClassNotFoundException: javassist.ClassPath PluginClassLoader(plugin=PluginDescriptor(name=GitPrefix, id=com.github.syuchan1005.emojiprefix, descriptorPath=plugin.xml, path=E:\JetBrains-ToolBox\apps\IDEA-U\ch-0\212.5284.40.plugins\GitPrefix.jar, version=2.1.4, package=null), packagePrefix=null, instanceId=28, state=active)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:235)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	... 28 more
```

**Screenshots**

**Desktop**
- IDE: (ex. IntelliJ IDEA Community 2018.X Build #IC-XXXXXXXXX)
- JRE: (ex. Java 1.8.0_XXX)
- JVM: (ex. OpenJDK 64-Bit Server VM by JetBrains s.r.o)
- Platform: (ex. Windows10, macOS Mojave, Ubuntu 18.04, etc....)

**.gitprefix file**
```text
```

**Comment by syuchan1005** [comment]
The `GitInjectorManager` is shown in stack trace. But, that class has been removed in GitPrefix v2.2.0. Could you update plugin to latest, and try it again?

**Comment by way-zer** [comment]
Do not know why, I can only see 2.1.4 in IDEA. But can see 2.2.0 in https://plugins.jetbrains.com/

**Comment by way-zer** [comment]
> Do not know why, I can only see 2.1.4 in IDEA. But can see 2.2.0 in https://plugins.jetbrains.com/

![image](https://user-images.githubusercontent.com/15688938/135488055-26362e00-61de-4ff2-b40a-51521457b604.png)

The support version not include IDEA 2021.2.2

**Comment by syuchan1005** [comment]
Sorry for the late reply. I released v2.2.1. please try it.
